<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerSearchPopup</class>
 <widget class="QWidget" name="ViewerSearchPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>1131</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>849</width>
    <height>1131</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true" />
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="popupCont" native="true">
       <property name="minimumSize">
        <size>
         <width>530</width>
         <height>607</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>750</width>
         <height>858</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="headerPopup">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>147</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>208</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>57</width>
                 <height>28</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="closeBtn">
               <property name="text">
                <string />
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>57</width>
                 <height>28</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="MyQLineEdit" name="searchLineEdit">
               <property name="text">
                <string>Enter text to search here...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearSearchBtn">
               <property name="text">
                <string />
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>113</width>
                 <height>28</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="resultsNumLbl">
             <property name="text">
              <string>60 resultados</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="ViewerSearchResultItem" name="SearchResultItem1" native="true" />
        </item>
        <item>
         <widget class="ViewerSearchResultItem" name="SearchResultItem2" native="true" />
        </item>
        <item>
         <widget class="ViewerSearchResultItem" name="SearchResultItem3" native="true" />
        </item>
        <item>
         <widget class="ViewerSearchResultItem" name="SearchResultItem4" native="true" />
        </item>
        <item>
         <widget class="ViewerSearchResultItem" name="SearchResultItem5" native="true" />
        </item>
        <item>
         <widget class="QWidget" name="processSearchCont" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>57</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="processLbl">
             <property name="text">
              <string>Procesando los datos ...</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>57</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ViewerVerticalPagerPopup" name="VerticalPagerPopup" native="true">
          <property name="minimumSize">
           <size>
            <width>540</width>
            <height>60</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="arrowCont" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <widget class="QLabel" name="bottomArrowLbl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string />
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="expandingViewerPopup">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyQLineEdit</class>
   <extends>QLineEdit</extends>
   <header>MyQLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ViewerVerticalPagerPopup</class>
   <extends>QWidget</extends>
   <header>ViewerVerticalPagerPopup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewerSearchResultItem</class>
   <extends>QWidget</extends>
   <header>ViewerSearchResultItem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources />
 <connections />
</ui>